docs/development/shareable-configs.md

```diff
@@ -19,6 +19,13 @@ In human-understandable form, the rules are:
 - If a package name is specified and has no preset name, then `default` is assumed, e.g. `@rarkins` expands in full to `@rarkins/renovate-config:default` and `rarkins` expands in full to `renovate-config-rarkins:default`
 - There is a special "default" namespace where no package name is necessary. e.g. `:webapp` (not the leading `:`) expands to `renovate-config-default:webapp`
 
+## Onboarding PR behavior
+
+When Renovate creates a onboarding PR, it tries to find any global Renovate configuration in the namespace of the onboarding repository.
+
+If the bot finds a global Renovate bot configuration, it will put a link to this repository in the `extends` array of the onboarding pull request.
+If the bot does not find a global configuration, it will create the default Renovate configuration.
+
```

Comment:
How about a simpler text like this?

It's really hard for me to find the right words to describe the behavior in full. Also if I describe the behavior in full, I'm basically duplicating the code.

If the ins and outs of this behavior is really important, we should link to the code maybe? That way developer types can figure out how it works from that, and normal users can just get the general idea that Renovate bot tries to help you apply your global renovate config. 😄

Reply:
We should at least document the two repos and the corresponding files which will be used.

Reply:
> We should at least document the two repos and the corresponding files which will be used.

I agree, but I don't know how to put that into a proper explanation myself... 🙈

@rarkins Are you willing to write a rough draft? I'm stuck on figuring out how to explain the behavior, and I'm still a bit fuzzy on the behavior itself.